lib/src/base-styles/_nx-colors.scss

```diff
@@ -554,6 +570,22 @@
     --nx-color-status-indicator-error-icon-background: var(--nx-swatch-red-40);
     --nx-color-status-indicator-error-icon-border: var(--nx-color-status-indicator-error-icon-background);
 
+    // NxGlobalSidebar
+    --nx-color-global-sidebar-background: var(--nx-color-component-background);
+    --nx-color-global-sidebar-separator-color: var(--nx-swatch-indigo-25);
+    --nx-color-global-sidebar-menu-icon-border-focus: var(--nx-color-interactive-border-focus);
+    --nx-color-global-sidebar-menu-icon-color: var(--nx-swatch-indigo-10);
+    --nx-color-global-sidebar-menu-icon-background: var(--nx-color-interactive-border-focus);
```

Comment:
> Should this just use the swatch instead of `--nx-color-interactive-border-focus`? It seems slightly strange to base the _hover_ _background_ color on the _focus_ _border_ color.

Makes sense. Changed here: 892372fe23a6d0bff494f6229c353d2a76986cb3